runtime/lib/runtime.cpp

```diff
@@ -176,7 +197,7 @@ size_t getNumAvailableDevices() {
     return ::tt::runtime::ttmetal::getNumAvailableDevices();
   }
 #endif
-  throw std::runtime_error("runtime is not enabled");
+  LOG_FATAL("runtime is not enabled");
```

Comment:
Next time for a big change, let's do cleanup/rename stuff like this as a separate PR, it makes reviewing the core change harder.